SPECS/python3/python3.spec

```diff
@@ -1,34 +1,22 @@
 %global openssl_flags -DOPENSSL_NO_SSL3 -DOPENSSL_NO_SSL2 -DOPENSSL_NO_COMP
 %global __brp_python_bytecompile %{nil}
-# Automating the extraction of these alternate version strings has proven to be tricky,
-# with regards to tooling available in the toolchain build environment.
-# These will be manually maintained for the time being.
-%global majmin 3.9
-%global majmin_nodots 39
-# See Lib/ensurepip/__init__.py in Source0 for these version numbers
-%global pip_version 22.0.4
-%global setuptools_version 58.1.0
+%define majmin %(echo %{version} | cut -d. -f1-2)
+%define majmin_nodots %(echo %{majmin} | tr -d .)
+# See Lib/ensurepip/__init__.py in Source0 for the pip version number
+%global pip_version 23.2.1
 
 Summary:        A high-level scripting language
 Name:           python3
-Version:        3.9.14
-Release:        8%{?dist}
+Version:        3.12.0
+Release:        1%{?dist}
 License:        PSF
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          System Environment/Programming
 URL:            https://www.python.org/
 Source0:        https://www.python.org/ftp/python/%{version}/Python-%{version}.tar.xz
+Source1:        pathfix.py
```

Comment:
Could we add a comment why this is included separately? I see our specs rely on this script and I'm guessing it's just not part of the new version of Python, but this comment may be useful in the future to understand why this file is special.

Reply:
Sure, I can add a comment. But basically, it was removed from the python 3.12 source (and previously available in python 3.9 source), and there is still code that depends on it and breaks without it.